packages/oaf/src/commands/_player.ts

```diff
@@ -0,0 +1,60 @@
+import type { Prisma } from "@prisma/client";
+
+import { prisma } from "../clients/database.js";
+import { kolClient } from "../clients/kol.js";
+
+export function validPlayerIdentifier(identifier: string) {
+  // If a player id: a number!
+  // If a username: 3 to 30 alphanumeric characters, starting with alpha, may contain underscores or spaces
+  return /^([a-zA-Z][a-zA-Z0-9_ ]{2,29})|[0-9]+$/.test(identifier);
```

Comment:
Would it be good to split the regex into several for readability?
Also it does not seem to allow Discord handles which the code below parses.